### PR TITLE
internet notification

### PR DESCRIPTION
**Pull request by katendejericho5**
#### Summary of Changes (What does this PR do?)

- Displays a banner everytime  ther is no internet connection

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

**Review by NoahNsimbe — COMMENTED** [review]
Thanks @katendejericho5, but the changes don't seem to match what is on figma.

File: mobile/lib/screens/offline_banner.dart [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                                  color: Colors.white,
+                                  fontSize: 12,
+                                ),
+                              ),
+                            ),
+                            const SizedBox(
+                              width: 55,
+                            ),
+                            Container(
+                              color: const Color.fromARGB(0, 0, 0, 0),
```
Comment (NoahNsimbe) [inline mobile/lib/screens/offline_banner.dart:92]: What colour is this?? @katendejericho5

File: mobile/lib/screens/offline_banner.dart [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                              alignment: Alignment.center,
+                              child: Text(
+                                AppLocalizations.of(context)!.dismiss,
+                                textAlign: TextAlign.center,
+                                style: const TextStyle(
+                                  color: Color.fromARGB(255, 255, 255, 255),
```
Comment (NoahNsimbe) [inline mobile/lib/screens/offline_banner.dart:98]: Same here, what colour is this?

File: mobile/lib/screens/offline_banner.dart [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            Positioned(
+              top: 0,
+              left: 0,
+              right: 0,
+              child: Visibility(
+                visible: isVisible,
+                child: SizedBox(
+                  height: 50,
+                  child: Container(
+                    color: const Color.fromARGB(195, 244, 67, 54),
```
Comment (NoahNsimbe) [inline mobile/lib/screens/offline_banner.dart:61]: Same here, what colour is this??

File: mobile/lib/screens/offline_banner.dart [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                    child: Padding(
+                      padding: const EdgeInsets.fromLTRB(0, 25, 0, 0),
```
Comment (NoahNsimbe) [inline mobile/lib/screens/offline_banner.dart:63]: I think we use top padding.
`padding: const EdgeInsets.only(top: 25),`

File: mobile/lib/screens/settings/about_page.dart [hunk 5]
```diff
@@ -25,65 +27,67 @@ class _AboutAirQoState extends State<AboutAirQo> {
...
+                  style: CustomTextStyle.headline11(context),
+                ),
+                const SizedBox(
+                  height: 12,
+                ),
+                Text(
+                  _packageInfo.version,
                   style: Theme.of(context).textTheme.titleMedium?.copyWith(
-                        color: CustomColors.appColorBlue,
+                        color: CustomColors.appColorBlack.withOpacity(0.5),
```
Comment (NoahNsimbe) [inline mobile/lib/screens/settings/about_page.dart:59]: @katendejericho5 why have we made this change??

**Comment by Baalmart** [comment]
@katendejericho5 please resolve the merge conflicts in this PR

**Comment by katendejericho5** [comment]
> @katendejericho5 please resolve the merge conflicts in this PR

i have resolved them

**Comment by katendejericho5** [comment]
i have resolved all the pending issues @NoahNsimbe , @BenjaminSsempala , @Mnoble-19 , @Baalmart

**Review by NoahNsimbe — COMMENTED** [review]
Thanks, @katendejericho5. We might want to consider writing some widget tests because merging this PR will reduce the tests' coverage by 0.11%

**Comment by katendejericho5** [comment]
> Thanks, @katendejericho5. We might want to consider writing some widget tests because merging this PR will reduce the tests' coverage by 0.11%

@NoahNsimbe , i have added unit test for the Internetbanner cubit

**Review by katendejericho5 — COMMENTED** [review]
Merge conflicts are resolved

**Pull request merged**